src/usr/local/www/status_dhcp_leases.php

```diff
@@ -300,8 +300,9 @@ function remove_duplicate($array, $field) {
 				$slease['online'] = in_array(strtolower($slease['mac']), $arpdata_mac) ? $online_string : $offline_string;
 				$slease['staticmap_array_index'] = $staticmap_array_index;
 				$leases[] = $slease;
-				$staticmap_array_index++;
 			}
+
+			$staticmap_array_index++;
```

Comment:
I was wondering if foreach from line 284 couldn't be replaced by:
```
foreach ($config['dhcpd'][$ifname]['staticmap'] as $idx => $static) {
```
and then $idx would be our staticmap_array_index